pkg/cluster/cache_whitebox_test.go

```diff
@@ -425,12 +389,8 @@ func TestMultipleActionsInParallel(t *testing.T) {
 			go func() {
 				defer waitForFinished.Done()
 				latch.Wait()
-				clusters := clusterCache.getCachedToolchainClustersByType(clusterToTest.Type)
-				if len(clusters) == 1 {
-					assert.Equal(t, clusterToTest, clusters[0])
-				} else {
-					assert.Empty(t, clusters)
-				}
+				clusterCache.getCachedToolchainClusters()
+
```

Comment:
My initial understanding was different, i thought since we are not giving type, it will bring in all clusters and finding a single cluster out it was somehow confusing for me.. , but i get your point, adding it

Reply:
but we do require that generic func here as we are testing multiple actions in parallel, so removing this whole func wont work for the whole test, and keeping a check of length also doesnt work

Reply:
I fully agree with @mfrancisc, this test doesn't verify anything - it only calls the function, but doesn't do any assertion on the returned value

Reply:
okay shall remove it